### PR TITLE
Support building docs in pdf

### PR DESCRIPTION
**Pull request by F-park**
user can download docs in pdf in readthedocs

### CODE REVIEW

**Comment by gaogaotiantian** [comment]
This seems like a very random feature with no request. I don't think anyone is using pdf docs.

**Comment by F-park** [comment]
> This seems like a very random feature with no request. I don't think anyone is using pdf docs.

I need the pdf docs and I revert the unnecessary modify.

**Comment by gaogaotiantian** [comment]
This seems like a very odd request, but the code change is small. What's the result of this? Will it just work so you can download pdf from the readthedocs website?

**Comment by F-park** [comment]
It just a [build config](https://docs.readthedocs.io/en/stable/config-file/v2.html#formats), and I download pdf from the readthedocs website.

image

File: .readthedocs.yml [hunk 1]
```diff
@@ -19,6 +19,10 @@ sphinx:
 #mkdocs:
 #  configuration: mkdocs.yml
 
+# Optionally build your docs in additional formats such as PDF and ePub
```
Comment (gaogaotiantian) [inline .readthedocs.yml:22]:
```suggestion
# Enable pdf download on readthedocs
```

**Pull request merged**